app/models/imports/podcast_rss_import.rb

```diff
@@ -145,8 +145,7 @@ def build_podcast_attributes
     podcast_attributes[:explicit] = explicit(feed.itunes_explicit, "false")
     podcast_attributes[:new_feed_url] = clean_string(feed.itunes_new_feed_url)
     podcast_attributes[:enclosure_prefix] ||= enclosure_prefix(feed.entries.first)
-    podcast_attributes[:feedburner_url] ||= feedburner_url(feed.feedburner_name)
-    podcast_attributes[:url] ||= feedburner_url(feed.feedburner_name)
+    podcast_attributes[:url] ||= clean_string(feed.feed_url)
```

Comment:
I just nuked the "feedburner name" parsing code, which looks like it was meant to strip the slug from feedburner urls, and use it as the `podcast.path` in our system.  (We deprecated those string paths, and just use the podcast ID in the S3 paths now).

And we now parse/pull the "self" link href into this feed_url property (see the feedjira parser file).